src/builder.rs

```diff
@@ -624,6 +629,48 @@ impl CpModelBuilder {
         }
     }
 
+    /// Add all solution hints from a solution.
```

Comment:
providing a solution with a vector is very error prone, as you cannot really know the order of the variables.

Reply:
Ok